sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Checks that all tunnels are up
+#
+# Return CRITICAL if any tunnel is down
+#
+# Dean Daskalantonakis <ddaskal@us.ibm.com>
+# Carl Hannusch <channusch@us.ibm.com>
+
+import argparse
+import re
+import subprocess
+import sys
+
+STATE_OK = 0
+STATE_CRITICAL = 2
+BACKUP = 1
```

Comment:
Didnt mean to leave the BACKUP global there, I'll pull that out.  That was used in a different check that I used as a template for this one.